src/open_inwoner/templates/pages/product/detail.html

```diff
@@ -24,6 +24,8 @@
 {% endblock sidebar_content %}
 
 {% block content %}
+    {% with question_set=object.question_set.exists files=object.files.exists conditions=object.conditions.exists locations=object.locations.exists product_links_exist=product_links.exists published_related_products=object.related_products.published.exists contacts=object.contacts.exists %}
```

Comment:
I'm not convinced this is a good idea. For one the names are not great (storing booleans in things that are named like collections).

Also the amount of queries is a bit much maybe? 

If we're aliasing anyway then just alias the query so Django can cache the result and do both the the if/else and iteration on the same result (instead of doing .exists() everywhere or doing multiple queries for the same things).